cmd/zetaclientd/utils.go

```diff
@@ -199,5 +202,40 @@ func CreateChainObserverMap(
 		}
 	}
 
+	// FIXME_SOLANA: config chain params
```

Comment:
It also seems like we can complete this now by updating the chain params

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a